### ISSUE TITLE
Issue on individual deck screen

### ISSUE THREAD

**Issue by Brandon268** [my-Anki/Beautify-Anki#20]
### Pie charts not displayed on individual deck screen

See album: https://imgur.com/a/TAmt3UE

Windows 10 / Anki version 2.1.25

**Comment by keithcdodd** [comment]
Hey, I had the same exact issue, and I was confused because I even had the "correct review heatmap" so I thought it had to be something else. Turns out for me, uninstalling review heatmap and reinstalling it with a new download of the anki addon did the trick. If that doesn't do it, the only other change I made is I set the buttons to false in the config for Beautify (setting the buttons to false fixed some issue I was having with Speed Focus Mode addon and Automatic Scoring) and restarted anki after that.

For uninstalling and reinstalling review heatmap for what worked for me:
(1) Anki --> Tools --> Addons --> Delete Review Heatmap
(2) https://github.com/glutanimate/review-heatmap/releases --> Download latest release **review-heatmap-v1.0.0-beta.1-anki21.ankiaddon** (can be found hidden under Assets within Version 1.0.0-beta.1)
(3) Anki --> Tools --> Addons --> Install From File --> select your newly downloaded **review-heatmap-v1.0.0-beta.1-anki21.ankiaddon** 
(4) Restart Anki
(5) Hopefully it is fixed
